README.md

```diff
@@ -9,6 +9,12 @@ This library is still in development.
 If you would like to participate in the beta tests, please write to ai@kx.com. 
 
 
+## Requirements ##
+
+KDB+ >=3.5 64-bit
```

Comment:
Please specify which C compiler is required.  It looks like you use GNU extensions such as `typeof` which may not be universally available.